.github/workflows/pr_test.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     strategy:
       matrix:
         tinygo-version: [0.24.0]
-        go-version: [1.19.x]
+        go-version: [1.18.x]
```

Comment:
Previously I bumped this to use the latest go1.19 but realised that this was causing issues running the tests as there is a constraint on using go < 1.19.